semcore/scroll-area/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [5.34.0] - 2024-07-22
+
+### Added
+
+- Property `observeParentSize` to enable recalculation if the parent of ScrollArea is resized
```

Comment:
I guess you should also mention the default behavior change here

Reply:
added